tests/pds/dataservice_test.go

```diff
@@ -627,6 +628,88 @@ var _ = Describe("{UpgradeDataServiceImage}", func() {
 	})
 })
 
+var _ = Describe("{CordonNodeAndDeletePod}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("CordonNodeAndDeletePod", "Deploys a data service, cordons one selected node, deletes the pod from that node", pdsLabels, 0)
+	})
+
+	It("Cordon a Node and Delete Pods from that node", func() {
+		Step("Deploy, Validate, Cordon Node and Delete Pods, Validate new Pod, Validate Storage, Run Workload on Data Service", func() {
+			for _, ds := range params.DataServiceToTest {
+				// Run this Test on Consul only. We can enable more Data Services later from here if needed.
+				// Consul is a random choice. We can choose any other Data service as well.
+				// This test case is pretty much Data service independent
+				if ds.Name != consul {
+					log.Warnf("Skipping %v dataservice ", ds.Name)
+					continue
+				}
+				isDeploymentsDeleted = false
+				deployment, _, _, err = DeployandValidateDataServices(ds, tenantID, projectID)
+				log.FailOnError(err, "Error while deploying data services")
+				log.InfoD("Running Check Node DataService Func %v ", ds.Name)
+				nodes, err := pdslib.GetNodesOfSS(*deployment.ClusterResourceName, namespace)
+				log.FailOnError(err, "Cannot fetch nodes of the running Data Service")
+				for _, node := range nodes {
+					log.InfoD("Node Name is: %s", node.Name)
```

Comment:
you can add one log statement like

log.InfoD("Resource %s is deployed on nodes  %s", *deployment.ClusterResourceName,nodes)
if nodes is printing full object you can create another list with only names and print it

Reply:
Thanks - Removed this log line - was there just for testing purposes